notebooks/239-image-bind/239-image-bind-quantize-nncf.ipynb

```diff
@@ -0,0 +1,1570 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    
</pre><p>it maybe strange situation when you have no repo dir, but trying to get IR inside in cell below. Probably in this case the first notebook should executed before inside this or at least provide some error/warning that model is not converted yet? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1155/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>